snuba/web/bulk_delete_query.py

```diff
@@ -266,8 +269,8 @@ def delete_from_tables(
     tables: Sequence[str],
     conditions: Dict[str, Any],
     attribution_info: AttributionInfo,
+    attribute_conditions: Optional[AttributeConditions] = None,
```

Comment:
down below (line 277) don't we need to add the attributes to the query to get an accurate count of how many rows to delete?

> `query = construct_query(storage, table, _construct_condition(conditions))`

Reply:
yeah, that's a good call. I will make that change pending if Volo wants me to rearrange how we handle attribute conditions